pytorch_lightning/trainer/data_loading.py

```diff
@@ -6,44 +6,24 @@
 
 import torch
 import torch.distributed as torch_distrib
-from torch.utils.data import DataLoader, RandomSampler, SequentialSampler
+from torch.utils.data import DataLoader, RandomSampler, SequentialSampler, IterableDataset
 from torch.utils.data.distributed import DistributedSampler
 
 from pytorch_lightning.core import LightningModule
-from pytorch_lightning.utilities import rank_zero_warn
+from pytorch_lightning.utilities import rank_zero_warn, is_xla_available, is_horovod_available
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
 
-try:
-    from torch.utils.data import IterableDataset
-    ITERABLE_DATASET_EXISTS = True
-except ImportError:
-    ITERABLE_DATASET_EXISTS = False
```

Comment:
removed this, because iterable dataset is part of torch since 1.2 and we support torch >= 1.3